notebooks/Prediction for WaveRNN.ipynb

```diff
@@ -49,15 +50,7 @@
    "metadata": {
     "scrolled": true
    },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "pred text mel: 399 stop out: -5.445310592651367Stopping\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "sentence = 'Scientists at the CERN laboratory, say they have discovered a new particle.'\n",
```

Comment:
The. comma is misplaced here.

Reply:
In the sentence? That is the text it was predicted on.